### DISCUSSION TITLE
About description file and data

### DISCUSSION THREAD

**Discussion by Cervangirard** [ThinkR-open/fusen#37] [Ideas] [answered]
Dear maintainers

When we start a development with {fusen}, in the first rmd, it asks to complete the description file. Wouldn't it be better to put a 01_start in dev? like the golem package?

Also for the data part, it could be a parameter in the add_dev_history function. 

Why these interogations? Nowadays, with each new rmd, the description and data part comes back.

Thanks for this package !

**Comment by statnmap** [comment, marked as answer]
You can use the second template: `fusen::add_dev_history(name = "add")` in which case you get a light version of the first "dev_history.Rmd" file.

But I may need two starting templates. One for "first time {fusen} users" and one for "second time and others".
